solidity/createAdvancedOrder.sh

```
@@ -34,4 +35,5 @@ just create-advanced-order \
     "$factory_address" \
     "$rpc_url" \
     "$chain_id" \
+    "$mnemonic" \
```

Shellcheck (0.10.0)

[warning] mnemonic is referenced but not assigned.

(SC2154)